constructor/worker/worker.go

```diff
@@ -673,3 +682,85 @@ func FindCurrencyAmountWorker(rawInput string) (string, error) {
 		types.PrintStruct(input.Currency),
 	)
 }
+
+// LoadEnvWorker loads an environment variable and stores
+// it in state. This is useful for algorithmic fauceting.
+func LoadEnvWorker(rawInput string) (string, error) {
+	// We unmarshal the input here to handle string
+	// unwrapping automatically.
+	var input string
+	err := job.UnmarshalInput([]byte(rawInput), &input)
+	if err != nil {
+		return "", fmt.Errorf("%w: %s", ErrInvalidInput, err.Error())
+	}
+
+	return os.Getenv(input), nil
+}
+
+// HTTPRequestWorker makes an HTTP request and returns the response to
+// store in a variable. This is useful for algorithmic fauceting.
+func HTTPRequestWorker(rawInput string) (string, error) {
+	var input job.HTTPRequestInput
+	err := job.UnmarshalInput([]byte(rawInput), &input)
+	if err != nil {
+		return "", fmt.Errorf("%w: %s", ErrInvalidInput, err.Error())
+	}
+
+	if input.Timeout <= 0 {
+		return "", fmt.Errorf("%w: %d is not a valid timeout", ErrInvalidInput, input.Timeout)
+	}
+
+	if len(input.URL) == 0 {
+		return "", fmt.Errorf("%w: URL is empty", ErrInvalidInput)
+	}
```

Comment:
`url.Parse` can be a better way to validate URL
```suggestion
	if _, err := url.Parse(path); err != nil {
		return "", fmt.Errorf("%w: %s", ErrInvalidInput, err.Error())
	}
```

Reply:
Ooo very cool. Was not aware!

Reply:
I opted to go with `ParseRequestURI` instead...it is a lot more restrictive than `Parse`.